src/backend/distributed/citus--7.1-2--7.1-3.sql

```diff
@@ -0,0 +1,10 @@
+/* citus--7.1-2--7.1-3 */
+
+CREATE TABLE citus.pg_dist_metadata(
+    tag text NOT NULL,
```

Comment:
Would it make sense for tag to be a primary key?

Reply:
:+1: